pkg/parent/parent.go

```diff
@@ -77,12 +78,33 @@ func checkPreflight(opt Opt) error {
 	return nil
 }
 
+func createCleanupLock(sDir string, tempDir bool) error {
+	//lock state dir when using /tmp/ path
+	if tempDir {
+		stateDir, err := os.Open(sDir)
+		if err != nil {
+			return err
+		}
+		err = unix.Flock(int(stateDir.Fd()), unix.LOCK_SH)
+		if err != nil {
+			return errors.Errorf("failed to lock directory %s", sDir)
```

Comment:
can be just a warning rather than a critical error?

Reply:
yes for sure I will change that